src/views/Calendar.vue

```diff
@@ -1,13 +1,14 @@
 <template>
     <div>
         <CategoriesBar
+            :categories="categories"
             @prevPressed="$refs.calendar.prev()"
             @nextPressed="$refs.calendar.next()"
         />
         <v-calendar
             v-model="startTimeCalendar"
             class="calendar"
-            :events="calendarEvents"
+            :events="events.concat(holidays)"
```

Comment:
You can export that to a computed property

Reply:
computed doesnt recognize the class properties, so I created a new one instead, you can see it on the updated